src/MeetingsSDKAdapter.js

```diff
@@ -157,6 +160,44 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     }
   }
 
+  /**
+   * Fetches and returns a promise of a meeting title from a given destination
+   *
+   * @param {string} destination destination to start the meeting at
+   * @returns {string}
+   * @memberof MeetingsSDKAdapter
+   */
+  async fetchMeetingTitle(destination) {
+    const {id, type} = deconstructHydraId(destination);
+    let meetingTitle = destination;
+
+    if (type === HYDRA_ID_TYPE_PEOPLE) {
+      const {displayName} = await this.datasource.people.get(id);
+
+      meetingTitle = `${displayName}'s Personal Space`;
+    } else if (type === HYDRA_ID_TYPE_ROOM) {
+      const {title} = await this.datasource.rooms.get(id);
+
+      meetingTitle = title;
+    } else {
+      try {
+        const people = await this.datasource.people.list({destination});
+
+        if (!!people.items && people.items.length > 0) {
+          const [person] = people.items;
+          const {displayName} = person;
```

Comment:
```suggestion
          const {displayName} = people.items[0];
```

Maybe it's just me but the previous line confuses me a bit, is there an object called person?